src/prefect/engine/results/constant_result.py

```diff
@@ -0,0 +1,51 @@
+from typing import Any
+
+from prefect.engine.result import Result
+
+
+class ConstantResult(Result):
+    """
+    Hook for storing and retrieving constant Python objects. Only intended to be used
+    internally.
+
+    Args:
+        - value (Any): the underlying value that we wish to "handle"
+    """
+
+    def __init__(self, value: Any) -> None:
+        self.value = value
+        super().__init__(value=value)
+
+    def read(self, arg: str) -> Any:
+        """
+        Returns the underlying value regardless of the argument passed.
+
+        Args:
```

Comment:
```suggestion
```

Reply:
Updated the interface from when the change was suggested. Code was previously:
`arg: str`

In order to maintain the consistent interface with the parent `Result`, the interface has changed to make this optional.